src/request.rs

```diff
@@ -140,6 +140,12 @@ impl QueryParameterWrapper {
     }
 }
 
+impl QueryParameterWrapper {
+    pub(crate) fn map_to_row(row: Vec<&QueryParameterWrapper>) -> Vec<CqlValue> {
+        row.iter().map(|e| e.parameter.clone()).collect()
+    }
+}
```

Comment:
`map_to_row` - what does it mean? IIUC, it just clones the Vec, unwrapping and cloning the `CqlValue` on the way.

Reply:
I've updated this function name to something more (hopefully) meaningfull